### PR TITLE
Update Quick Start Guide

### PR DESCRIPTION
**Pull request by RukshiW**
Corrects https://github.com/wso2-enterprise/choreo/issues/14019

### CODE REVIEW

File: en/docs/get-started/quick-start-guide.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Follow this procedure to further develop the API by adding a new resource and si
      |-----------------|-----------------------|
      | **HTTP Method** | `GET` |
      | **Path** | `personalizedMessage` |
-     | **Return Type** | `string` |
+     | **Return Type** | `string|error` |
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:79]: The return type value is not getting reflected properly when viewing the file content.
Screenshot 2022-07-11 at 11 53 27

File: en/docs/get-started/quick-start-guide.md [hunk 2]
```diff
@@ -104,19 +103,50 @@ Follow this procedure to further develop the API by adding a new resource and si
...
+    ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
+    ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
    
-10. Click **Save**.
+ 9. Click **Save**.
 
 !!!info
-    You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can choose to work in the low-code view or pro-code view, or both simultaneously.    
+    You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can work in the low-code view or the pro-code view. You can also work in both views simultaneously.    
     ![Editor View](../assets/img/get-started/low-code-pro-code-side-by-side.png){.cInlineImage-full}
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:116]: Looks like the text on the left side has got selected by mistake on taking the screenshot. Shall we retake and add the screenshot?

File: en/docs/get-started/quick-start-guide.md [hunk 3]
```diff
@@ -104,19 +103,50 @@ Follow this procedure to further develop the API by adding a new resource and si
...
+
+5. Click **Execute**.
+
+    The Swagger view displays the response under **Response Body** as shown in the image below:
+
+    ![REST API response](../assets/img/get-started/rest-api-response.png){.cInlineImage-half}
+
+### Step 3: Commit Changes to GitHub
 
 Choreo maintains source code in a private GitHub repository. When you create a component, Choreo creates it in the private-GitHub repository associated with your account. Therefore, you must commit and push your changes if you want to change the API Implementation.
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:151]: We seem to have an unnecessary hyphen when referring to the private GitHub repository in the second sentence.

File: en/docs/get-started/quick-start-guide.md [hunk 4]
```diff
@@ -104,19 +103,50 @@ Follow this procedure to further develop the API by adding a new resource and si
...
+
+4. In the **name** field, enter any value (e.g., `Alice`).
+
+5. Click **Execute**.
+
+    The Swagger view displays the response under **Response Body** as shown in the image below:
+
+    ![REST API response](../assets/img/get-started/rest-api-response.png){.cInlineImage-half}
+
+### Step 3: Commit Changes to GitHub
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:149]: We should go with sentence-style here:
Commit changes to GitHub

File: en/docs/get-started/quick-start-guide.md [hunk 5]
```diff
@@ -104,19 +103,50 @@ Follow this procedure to further develop the API by adding a new resource and si
...
+   
+    ![Try REST API](../assets/img/get-started/try-rest-api.png){.cInlineImage-half}
+
+    This opens the Swagger view with details of your REST API.
+
+3. Expand the GET resource and click **Try it out**.
+
+    ![Try out REST API](../assets/img/get-started/try-out-rest-api.png){.cInlineImage-full}
+
+4. In the **name** field, enter any value (e.g., `Alice`).
```
Comment (Nashaath) [inline en/docs/get-started/quick-start-guide.md:141]: The Google style guide recommends the use of `for example` instead of `e.g.,`. Shall we go with that approach?
https://developers.google.com/style/word-list?hl=en#ecommerce

**Review by Nashaath — CHANGES_REQUESTED** [review]
Added a few minor suggestions.

File: en/docs/get-started/quick-start-guide.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/get-started/quick-start-guide.md]: First, I tried to replace this table with an html table to fix this, but it was not getting rendered, and the html code was showing instead of the table.

Then I tried to present this as a list of key-value pairs,. and did the same to the other tables to be consistent, but it was not looking good.

Felt that replacing each table row with sub-steps would make this guide too wordy.

Therefore, replaced only the affected table row with a sub step. This too seems a bit inconsistent.

Please share any other approaches you can think of to address this in a better way

File: en/docs/get-started/quick-start-guide.md (hunk 2 above)
Comment (RukshiW) [reply en/docs/get-started/quick-start-guide.md:119]: Done

File: en/docs/get-started/quick-start-guide.md (hunk 3 above)
Comment (RukshiW) [reply en/docs/get-started/quick-start-guide.md:154]: Fixed.

File: en/docs/get-started/quick-start-guide.md (hunk 4 above)
Comment (RukshiW) [reply en/docs/get-started/quick-start-guide.md]: Fixed

File: en/docs/get-started/quick-start-guide.md (hunk 5 above)
Comment (RukshiW) [reply en/docs/get-started/quick-start-guide.md]: Done

File: en/docs/get-started/quick-start-guide.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/get-started/quick-start-guide.md]: Fixed by adding a \ before the pipeline as suggested by Nisrin.

**Review by Nashaath — APPROVED** [review]
